src/assets/i18n/convert-to-csv.js

```diff
@@ -0,0 +1,218 @@
+const data = {
```

Comment:
Are we thinking of using this again? If we are I think it's fine to add

Reply:
oh whoops, didn't mean to add this in